### DISCUSSION TITLE
Dedikoducu kuşlar ve karga

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#44] [Beyin Fırtınası]
Küçük bir ormanda yaşayan kuşların tek eğlencesi dedikodu yapmakmış. Hatta bazı günler dedikodu yapmaktan avlanmayı ya da yiyecek toplamayı bile unutur olmuşlar. Bir süre sonra bunun böyle gidemeyeceğini anlamışlar. Bu sorunu çözse çözse bilge karga çözer deyip, karganın yuvasının yolunu tutmuşlar.

Bilge karga sorunu dinledikten sonra, “Çok kolay, dedikodu yapmayı bırakın, sorun kendiliğinden bitecektir” demiş. Bunu duyan diğer kuşlar “Aman bilge karga, biz dedikodusuz yaşayamayız. Kendimizden vazgeçtik, çocuklar için yardım istiyoruz senden. Ne olur hallet bunu.” demişler. Karga da “Hele bir anlatın bakalım, nasıl bir sorunmuş bu?” demiş ve kuşları dinlemeye başlamış.

“Başlangıçta her birimiz diğer kuşların bilmediği bilgilere sahip. Birbirimizin arkasından konuştuğumuzdan dedikodu her seferinde iki kuş arasında yapılıyor. Dedikodu bittiğinde iki kuş da birbirlerinin o ana kadar bildiği bütün bilgileri öğrenmiş oluyorlar ama hepimiz bütün dedikoduları öğrenmek istediğimizden bütün günümüz bir ona bir buna uçarak ve aynı dedikoduları defalarca dinleyerek geçiyor. Öyle bir yol bul ki yiyecek bulmak için bize daha çok zaman kalsın.”

Bunu duyan karga gülerek “E bundan kolay ne var? Hep beraber bir yerde toplanın ve dedikodunuzu topluca yapın, nasıl olsa akşam olduğunda herkes bütün dedikoduları öğrenmiyor mu?” demiş.

Kuşlar ise bu öneriye hep bir ağızdan itiraz etmiş. “Toplu yapamayız bilge karga, yanımızda bir üçüncü kuş bile olsa utanırız. Son ümidimiz sendin.” demişler.

Karga da pençesiyle gagasını kaşıyarak “Bunu bir düşüneyim. Yarın bu saatte bana gelin, çözümü size anlatayım” demiş.

Kuşlar da hemen bir çözüm bulunamamış olmasından ötürü hafif hayal kırıklığına uğramıs ama yine de umutlu bir şekilde “Tamam, yarın görüşürüz” deyip yuvalarına uçuşmuşlar.

 Karga acaba nasıl bir çözüm bulmuş?

**Comment by yigidi** [comment]
Bilge karga, kuşların dedikodu sorununa çözüm olarak aşağıdaki akıllıca yaklaşımı önerdi:

Karga, kuşlara toplu bir çözüm önerdiği zaman reddedildiğini görmüş ve kuşların utançları nedeniyle dedikodu yapamadığını fark etmişti. Bu nedenle, karga kuşlara bireysel olarak çözüm önerdi.

Her kuşun bir "dedikodu not defteri" oluşturmasını önerdi. Her gün, bir kuşun diğer kuş hakkında duyduğu dedikoduları not alması gerekiyordu. Bu defterler özel ve sadece kuşların kendileri tarafından görülebilecek şekilde tasarlanacaktı.

Ardından, her kuşun gün boyunca yaptığı notları "dedikodu paylaşım ağacı" adı verilen bir sistemde toplamalarını önerdi. Bu ağaçta, her kuş kendi notlarını ekleyecekti. Ancak, her kuş her gün yalnızca kendisinin bilmediği bilgileri ekleyebilirdi. Yani, bir kuşun bildiği bir dedikodu zaten ağaçta varsa, başka bir kuş bunu eklemeyecekti.

Bu şekilde, her kuşun notları zaman içinde birleşir ve kuşlar arasında dağılırken herkes bütün dedikoduları öğrenmiş olurdu. Kuşlar daha önce bildiği dedikoduları tekrar tekrar dinlemek zorunda kalmazdı çünkü her kuş zaten kendisi için yeni olan dedikoduları ekliyordu.

Sonuç olarak, bu çözüm sayesinde kuşlar hem utanç yaşamadan dedikodu yapabilirlerdi hem de dedikodu yapmaktan çok daha fazla vakit ayırarak yiyecek toplama ve avlanma gibi hayati ihtiyaçlarını karşılayabilirdi.
